client/components/jetpack/card/jetpack-product-card-alt-2/features-product-slide-out.tsx

```diff
@@ -31,40 +34,42 @@ const FeaturesProductSlideOut: FunctionComponent< Props > = ( {
 	productBillingTerm,
 	onProductClick,
 } ) => {
+	const { iconSlug, productSlug, displayName, description } = product;
+
 	const siteId = useSelector( getSelectedSiteId );
 	const currencyCode = useSelector( getCurrentUserCurrencyCode );
-	const translate = useTranslate();
+	const purchases = useSelector( ( state ) => getSitePurchases( state, siteId ) );
+	const sitePlan = useSelector( ( state ) => getSitePlan( state, siteId ) );
 
-	// Calculate the product price.
-	const { originalPrice, discountedPrice } = useItemPrice(
-		siteId,
-		product,
-		product?.monthlyProductSlug || ''
-	);
-	const isDiscounted = isFinite( discountedPrice );
-	const productPrice = isDiscounted ? discountedPrice : originalPrice;
-
-	const billingTimeFrame = durationToText( productBillingTerm );
+	const translate = useTranslate();
+	const { originalPrice } = useItemPrice( siteId, product, product?.monthlyProductSlug || '' );
```

Comment:
We should be getting the discountPrice also.  All the slide-outs are showing Monthly prices (Even when Yearly is selected). For example, Jetpack Backup (yearly) is $8(discounted), and Jetpack Backup (monthly) is $10.  The way it is showing here is Jetpack Backup is $10 for both monthly & yearly.

Reply:
Absolutely! I don't know why, in my mental modal `originalPrice` included both prices. Thanks for catching this :)